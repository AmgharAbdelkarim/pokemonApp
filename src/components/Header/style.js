import styled from "styled-components";

export const HeaderWrapper = styled.div`
  padding: 20px;
  border-bottom: 1px solid #e7e7e7;
  border-radius: 8px;
  background-color: #ffffff;
  box-shadow: 0 1px 3px 0 rgba(0, 0, 0, 0.25);
  > a {
    text-decoration: none;
  }
  > span {
    color: #04336b;
    font-size: 18px;
    font-weight: bold;
    letter-spacing: 0;
    line-height: 23px;
  }
`;
