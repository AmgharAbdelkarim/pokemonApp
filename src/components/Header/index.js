import {HeaderWrapper} from 'components/Header/style';
import React from 'react';
const Header = () => {
  return (
    <HeaderWrapper id="header">
      <a href="/">
        <span>poke App</span>
      </a>
    </HeaderWrapper>
  );
};

export default Header;
