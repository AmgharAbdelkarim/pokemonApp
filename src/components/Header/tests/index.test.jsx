import { cleanup, render } from "@testing-library/react";

import Header from "components/Header";

afterEach(cleanup);

describe("Header", () => {
  it("should Header render correctly", () => {
    const { asFragment, getByText } = render(Header());
    expect(getByText("poke App")).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
