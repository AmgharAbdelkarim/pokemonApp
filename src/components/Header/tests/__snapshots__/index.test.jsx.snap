// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should Header render correctly 1`] = `
<DocumentFragment>
  <div
    class="sc-bdfBwQ VIZhq"
    id="header"
  >
    <a
      href="/"
    >
      <span>
        poke App
      </span>
    </a>
  </div>
</DocumentFragment>
`;
